context/datapath.go

```diff
@@ -534,37 +476,19 @@ func (dpNode *DataPathNode) ActivateUpLinkPdr(smContext *SMContext, defQER *QER,
 
 	curULTunnel := dpNode.UpLinkTunnel
 	for name, ULPDR := range curULTunnel.PDR {
-		ULDestUPF := curULTunnel.DestEndPoint.UPF
 		ULPDR.QER = append(ULPDR.QER, defQER)
 
 		// Set Default precedence
 		if ULPDR.Precedence == 0 {
 			ULPDR.Precedence = defPrecedence
 		}
 
-		var iface *UPFInterfaceInfo
```

Comment:
What was motivation to delete this code?

Reply:
iface is still defined, but only in the scope where it is used. You can see `iface := nextULTunnel.DestEndPoint.UPF.GetInterface(models.UpInterfaceType_N9, smContext.Dnn)
` on line 518